### ISSUE TITLE
Спавнится очень мало бибов (60 FPS)

### ISSUE THREAD

**Issue by kkartman** [KranX/Vangers#277]
При включении 60 кадров, жуков-бибов на поверхности мира спавнится очень мало, максимум один-два штуки на экране.

**Comment by Aidoneus** [comment]
Да, тоже такое ощущение есть. Раньше стабильно прямо было - остановишься где-нибуд, и через полминутки уже целая стайка к тебе на экран выползает. А тут можно циклами стоять и не увидеть ни одного биба. Или одного-двух, обколесив пол-мира. Больше двух за раз на экране никогда не видел...

**Comment by ESandu** [comment]
Подтверждаю, табутаск на "срочно собери 10 бибов до конца цикла" мне не удалось выполнить даже с 5 попыток.

**Comment by stalkerg** [comment]
Сейчас в alpha колличество бибов такое же как и в мастер ветке. Если встать то 1-3 всегда гуляют где то.
